_posts/2017-11-27-Babel-7-and-TypeScript.markdown

```diff
@@ -0,0 +1,261 @@
+---
+layout: post
+title: "Babel 7 + TypeScript"
+date: 2017-11-27 14:18
+comments: true
+author: chris
+github-url: https://www.github.com/damassi
+twitter-url: http://twitter.com/damassi
```

Comment:
I think these two lines aren't needed anymore, as they are defined in `_config.yml`: 

https://github.com/artsy/artsy.github.io/blob/a3ae2942b9f1300854240ce336f5c0e8b91e2f65/_config.yml#L154-L157

Reply:
Yep

Reply:
I'd also move it to the `layout: epic`, personally